doc/menu_lua_api.txt

```diff
@@ -98,74 +98,86 @@ core.file_open_dialog(formname,caption)
 * core.get_game(index)
-	* returns:
-
-		{
-			id               = <id>,
-			path             = <full path to game>,
-			gamemods_path    = <path>,
-			name             = <name of game>,
-			menuicon_path    = <full path to menuicon>,
-			author           = "author",
-			DEPRECATED:
-			addon_mods_paths = {[1] = <path>,},
-		}
+    * returns:
+
+        {
+            id               = <id>,
+            path             = <full path to game>,
+            gamemods_path    = <path>,
+            name             = <name of game>,
+            menuicon_path    = <full path to menuicon>,
+            author           = "author",
+            DEPRECATED:
+            addon_mods_paths = {[1] = <path>,},
+        }
 
 * core.get_games() -> table of all games in upper format (possible in async calls)
 * core.get_content_info(path)
-	* returns
+    * returns
 
-		{
-			name             = "name of content",
-			type             = "mod" or "modpack" or "game" or "txp",
-			description      = "description",
-			author           = "author",
-			path             = "path/to/content",
-			depends          = {"mod", "names"}, -- mods only
-			optional_depends = {"mod", "names"}, -- mods only
-		}
+        {
+            name             = "name of content",
+            type             = "mod" or "modpack" or "game" or "txp",
+            description      = "description",
+            author           = "author",
+            path             = "path/to/content",
+            depends          = {"mod", "names"}, -- mods only
+            optional_depends = {"mod", "names"}, -- mods only
+        }
 
 
 Favorites:
-core.get_favorites(location) -> list of favorites (possible in async calls)
-^ location: "local" or "online"
-^ returns {
-	[1] = {
-	clients       = <number of clients/nil>,
-	clients_max   = <maximum number of clients/nil>,
-	version       = <server version/nil>,
-	password      = <true/nil>,
-	creative      = <true/nil>,
-	damage        = <true/nil>,
-	pvp           = <true/nil>,
-	description   = <server description/nil>,
-	name          = <server name/nil>,
-	address       = <address of server/nil>,
-	port          = <port>
-	},
+- core.get_favorites(location) -> list of favorites (possible in async calls)
+  - `location`: "local" or "online"
+  - returns {
+    [1] = {
+    clients       = <number of clients/nil>,
+    clients_max   = <maximum number of clients/nil>,
+    version       = <server version/nil>,
+    password      = <true/nil>,
+    creative      = <true/nil>,
+    damage        = <true/nil>,
+    pvp           = <true/nil>,
+    description   = <server description/nil>,
+    name          = <server name/nil>,
+    address       = <address of server/nil>,
+    port          = <port>
+    },
 }
-core.delete_favorite(id, location) -> success
+
+- core.add_favorite(server)
+  - Adds a server to the local favorites list
+  - `server`: {
+        `name` (server name)
+        `address`
+        `port`
+        `description`
```

Comment:
This is no real table and will cause confusion.
One might think that it's a list (eg. `{"mycoolserver", "bla.net",30000,"come and join"}` but the commas are missing.

It should be something like:
```
{
  name    = <server name>,
  address = <address of the server>,
  ...
}
```